deployment/contour/03-envoy.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    app: envoy
+  name: envoy
+  namespace: gimbal-contour
+spec:
+  selector:
+    matchLabels:
+      app: envoy
+  template:
+    metadata:
+      annotations:
+        prometheus.io/format: prometheus
+        prometheus.io/path: /stats
+        prometheus.io/port: "9001"
+        prometheus.io/scrape: "true"
+      labels:
+        app: envoy
+    spec:
+      containers:
+      - args:
+        - -c
+        - /config/contour.yaml
+        - --service-cluster
+        - cluster0
+        - --service-node
+        - node0
+        command:
+        - envoy
+        image: docker.io/envoyproxy/envoy-alpine:v1.6.0
+        imagePullPolicy: IfNotPresent
+        name: envoy
+        ports:
+        - containerPort: 80
+          hostPort: 80
+          name: http
+          protocol: TCP
+        - containerPort: 443
+          hostPort: 443
+          name: https
+          protocol: TCP
+        volumeMounts:
+          - name: contour-config
+            mountPath: /config
+      dnsPolicy: ClusterFirst
+      hostNetwork: true
+      initContainers:
+      - args:
+        - bootstrap
+        - /config/contour.yaml
+        - --admin-address
+        - 0.0.0.0
+        - --xds-address
+        - $(CONTOUR_SERVICE_HOST)
+        - --xds-port
+        - $(CONTOUR_SERVICE_PORT)
+        command:
+        - contour
+        image: gcr.io/heptio-images/contour:v0.5.0
+        imagePullPolicy: IfNotPresent
+        name: envoy-initconfig
+        volumeMounts:
+        - name: contour-config
+          mountPath: /config
+      volumes:
+      - name: contour-config
+        emptyDir: {}
+      restartPolicy: Always
```

Comment:
Should we disable automount of service account?

Reply:
The `contour-config`? That's setup by the initContainer.

Reply:
Sorry. Meant setting `automountServiceAccountToken: false` so that the default service account does not get mounted to the envoy pod, which does not need access to the API server.